deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -583,6 +583,10 @@ spec:
                               type: string
                             lastConnectionVersion:
                               type: integer
+                            startScriptStdoutMessage:
+                              type: string
```

Comment:
We can also declare maxLength for both these properties in the schema here, to advertise that to the client as well as to prevent us from violating that length ourselves (if some bug happens). Note that this will absolutely prevent us from going past that length, which could affect our truncation strategy... I'll have more comments about that elsewhere I think.

Reply:
I added `MaxLogLinesDump` role property (and default property) to specify the quantity of tail lines should be dumped to the `MemberStatusState` struct fields

Reply:
I'm not sure this should be app-definable. We have limited space in the resource document (K8s API accepts up to 3 MB, etcd backend by default can store up to 1 MB) so by allowing the kdapp author to define some field size we're opening the door for them to break certain sizes of kdcluster.

Reply:
So I guess my opinion is that KD should always have some hard character limit for these fields, regardless of anything else. We could support some fancier processing to try to show whole lines and/or allow the kdapp to specify some (lower) limit, but our own hard limit should be an unbreakable backstop.

Reply:
Well, I changed `MaxLogLinesDump` to `MaxLogSizeDump` field and related logic, that means the user may strongly limitate the characters quantity to dump. Also I added the logic to show whole lines only are included in this character range.